test/components/labelTests.ts

```diff
@@ -1,198 +1,240 @@
-    assert.isTrue((<any> label)._element.classed("label"), "title element has label css class");
-    assert.isTrue((<any> label)._element.classed("title-label"), "title element has title-label css class");
-    let textChildren = content.selectAll("text");
-    assert.lengthOf(textChildren, 1, "There is one text node in the parent element");
-
-    let text = content.select("text");
-    let bbox = Plottable.Utils.DOM.elementBBox(text);
-    assert.closeTo(bbox.height, label.height(), 0.5, "text height === label.minimumHeight()");
-    assert.strictEqual(text.node().textContent, "A CHART TITLE", "node's text content is as expected");
-    svg.remove();
-  });
-
-  it("angle() error-checking", () => {
-    let label360 = new Plottable.Components.Label("noScope", 360);
-    assert.strictEqual(label360.angle(), 0, "angles are converted to range [-180, 180] (360 -> 0)");
-    let label270 = new Plottable.Components.Label("turnRight", 270);
-    assert.strictEqual(label270.angle(), -90, "angles are converted to range [-180, 180] (270 -> -90)");
-    let labelNeg270 = new Plottable.Components.Label("turnRight", -270);
-    assert.strictEqual(labelNeg270.angle(), 90, "angles are converted to range [-180, 180] (-270 -> 90)");
-    let badAngle = 10;
-    assert.throws(() => new Plottable.Components.Label("foo").angle(badAngle), Error);
-    assert.throws(() => new Plottable.Components.Label("foo", badAngle), Error);
-  });
-
-  it("Left-rotated text is handled properly", () => {
-    let svg = TestMethods.generateSVG(100, 400);
-    let label = new Plottable.Components.AxisLabel("LEFT-ROTATED LABEL", -90);
-    label.renderTo(svg);
-    let content = (<any> label)._content;
-    let text = content.select("text");
-    let textBBox = Plottable.Utils.DOM.elementBBox(text);
-    TestMethods.assertBBoxInclusion((<any> label)._element.select(".bounding-box"), text);
-    assert.closeTo(textBBox.height, label.width(), window.Pixel_CloseTo_Requirement, "text height");
-    assert.closeTo(textBBox.width, label.height(), window.Pixel_CloseTo_Requirement, "text width");
-    svg.remove();
-  });
-
-  it("Right-rotated text is handled properly", () => {
-    let svg = TestMethods.generateSVG(100, 400);
-    let label = new Plottable.Components.AxisLabel("RIGHT-ROTATED LABEL", 90);
-    label.renderTo(svg);
-    let content = (<any> label)._content;
-    let text = content.select("text");
-    let textBBox = Plottable.Utils.DOM.elementBBox(text);
-    TestMethods.assertBBoxInclusion((<any> label)._element.select(".bounding-box"), text);
-    assert.closeTo(textBBox.height, label.width(), window.Pixel_CloseTo_Requirement, "text height");
-    assert.closeTo(textBBox.width, label.height(), window.Pixel_CloseTo_Requirement, "text width");
-    svg.remove();
+  const LABEL_CLASS = "label";
+  let CLOSETO_REQUIRMENT;
+  before(() => {
+    CLOSETO_REQUIRMENT = window.Pixel_CloseTo_Requirement;
+    window.Pixel_CloseTo_Requirement = 2;
```

Comment:
Why the replacement? Can we just use `const CLOSETO_REQUIREMENT = 2` instead?

We should also tag it with a `HACKHACK` that points to https://github.com/palantir/plottable/issues/2422, since that's probably the root cause of the problem here.